modules/social_features/social_event/config/install/core.entity_view_display.node.event.activity.yml

```diff
@@ -90,5 +90,6 @@ hidden:
   field_files: true
   like_and_dislike: true
   groups: true
+  groups_type_public_group: true
```

Comment:
This seems like a change that doesn't belong in this PR?

Same for all files where this line was added.

Reply:
Should be in the public group PR. I do not see a problem by fixing a few small issues here and there without making separate tickets & PRs for everything. But feel free to separate it, the change should be self evident.